ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -97,6 +97,7 @@ def test_numpy_nanmean(
         keepdims=keep_dims,
         where=where,
         test_values=False,
+        rtol=1e-2,
```

Comment:
This should not be passed in at the end pf the function, kindly look at the structure of the functions that how other functions are implemented and then pass it there. :slightly_smiling_face: